### PR TITLE
add play/pause button to device

### PR DESCRIPTION
**Pull request by imaitland**
Basic PR adding play/pause functionality to the device, following PR will hook it up into the /abort endpoint.

open to feedback for icons and language...

### CODE REVIEW

File: .github/workflows/audit.yml
Comment (imaitland) [inline .github/workflows/audit.yml:1]: Removed the npm audit step. With dependabot it is redundant.

File: package-lock.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
       },
       "devDependencies": {
         "@playwright/test": "^1.45.3",
```
Comment (jamienoss) [inline package-lock.json:35]: Are changes in this file needed/expected?

File: package.json [hunk 2]
```diff
@@ -44,8 +44,8 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.45.3",
-    "@remix-run/dev": "^2.10.2",
-    "@remix-run/testing": "^2.10.3",
+    "@remix-run/dev": "^2.13.1",
+    "@remix-run/testing": "^2.13.1",
```
Comment (jamienoss) [inline package.json:48]: This doesn't look right, right? Looks like you accidentally downgraded these packages maybe?

**Review by jamienoss — COMMENTED** [review]
* I have no idea what I'm looking at to review the ``.tsx`` file.
 * Looks like you might have possible left in some dependency requirements files that shouldn't have? 🤷 
 * I like the look of the button though from the vid-snippet you posted in the description.
 
 I would have approved if it weren't for the ``package.json`` and ``package-lock.json``.

File: package.json (hunk 2 above)
Comment (imaitland) [reply package.json:48]: Upgraded `npm upgrade` will upgrade packages.

File: package-lock.json (hunk 1 above)
Comment (imaitland) [reply package-lock.json:35]: yep, package-lock is generated by running `npm install` it pins or "freezes" the dependency tree.

**Comment by jamienoss** [comment]
@imaitland & @amitschang Looking at https://github.com/ssec-jhu/evolver-ng/pull/200, I would think the UI needs a safety measure to account for someone hitting "play" when something is already running. Otherwise, it'll clobber everything (e.g., config on file) and restart.

**Comment by imaitland** [comment]
> * I have no idea what I'm looking at to review the `.tsx` file.
> 
>     * Looks like you might have possible left in some dependency requirements files that shouldn't have? 🤷
> 
>     * I like the look of the button though from the vid-snippet you posted in the description.
> 
> 
> I would have approved if it weren't for the `package.json` and `package-lock.json`.

Do my comments regarding changes to package.json and package-lock.json make sense? Happy to break them out into a seperate pr, tho i don't think this is necessary here.

**Comment by imaitland** [comment]
> @imaitland & @amitschang Looking at [ssec-jhu/evolver-ng#200](https://github.com/ssec-jhu/evolver-ng/pull/200), I would think the UI needs a safety measure to account for someone hitting "play" when something is already running. Otherwise, it'll clobber everything (e.g., config on file) and restart.

@jamienoss what specifically does it clobber? 

My understanding is that sending a `POST` request to `/stop` does the following:

1. toggle config flags: 
    config.enable_control = False
    config.enable_commit = False
2. Save config to evolver.yml

so nothing should be clobbered (except those flags) by stopping and then starting the evolver?

**Review by jamienoss — APPROVED** [review]

**Comment by jamienoss** [comment]
Re: https://github.com/ssec-jhu/evolver-ui/pull/5#issuecomment-2450551893

That's correct, but only if you abort first. It would be a shame to accidentally click ``start`` whilst an experiment is already running and thus restart it.

**Comment by imaitland** [comment]
> Re: [#5 (comment)](https://github.com/ssec-jhu/evolver-ui/pull/5#issuecomment-2450551893)
> 
> That's correct, but only if you abort first. It would be a shame to accidentally click `start` whilst an experiment is already running and thus restart it.

start/stop as implemented in this pr is a toggle. So from a UX pov - you can't start an already running evolver.

User can do whatever they want against the http api.

If there needs to be a guard it should be on the device's `/start` http endpoint, if there isn't one already - i.e. make it impossible to start an already running evovler.

**Comment by jamienoss** [comment]
>start/stop as implemented in this pr is a toggle. So from a UX pov - you can't start an already running evolver.

Awesome! 👍

**Pull request merged**